### ISSUE TITLE
Datamap containing a missing line critical error

### ISSUE THREAD

**Issue by banillie** [yulqen/datamaps#20]
Recent testing of datamaps export master command raised the following error
```
CRITICAL - Datamap contains a missing line. Please fix datamap before proceeding.
```
was able to resolve this issue by deleting rows (~10) from the bottom of the datamap. Therefore likely issue with datamaps calculating last row in dm.cvs. Will send dm.cvs file causing error to you for testing. cmd is `datamaps export master [PATH to master]` and dm.cvs file in datamaps\input file. Any dummy master can be used.

**Comment by yulqen** [comment]
...have asked on Teams to be emailed a copy of the offending CSV file (to personal email please). Thank you!

**Comment by banillie** [comment]
done.

**Comment by yulqen** [comment]
The offending file contains thousands of empty rows. I will need to include a check for this is in the suite of checks that take place on a datamap prior to processing.

**Comment by yulqen** [comment]
Fixed in v1.1.5.
